ebos/eclinterregflows.cc

```diff
@@ -19,6 +19,10 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif // HAVE_CONFIG_H
```

Comment:
It obviously doesn't hurt and I appreciate the consistency.  It also doesn't help because there's nothing in this file that depends on configuration parameter.